client/assets/stylesheets/_nav-unification.scss

```diff
@@ -57,7 +48,7 @@
 	.masterbar__toggle-drafts.button.is-borderless {
 		height: 24px;
 		margin: 4px 8px;
-		border-radius: 5px;
+		border-radius: 2px;
```

Comment:
What's this change for?

Reply:
I tried to comply with https://github.com/Automattic/wp-calypso/blob/12df4ecc07993c51c549616de6a1c93ce7d95077/.stylelintrc#L6

But, I just thought that this has to comply with `wp-admin` so I reverted it https://github.com/Automattic/wp-calypso/pull/46035/commits/7014168217b5798fc5d03b54442b33080e639e6e

Good catch, thanks.